doc/install_mne_c.rst

```diff
@@ -2,290 +2,162 @@
 
 .. _install_mne_c:
 
-Install MNE-C
--------------
+Installing MNE-C
```

Comment:
this document changed a lot.  I went with the "make bold edits" approach in an effort to make the page substantially shorter.  Happy to discuss further / put some things back, if there's opposition.